go.mod

```diff
@@ -1,30 +1,31 @@
 module github.com/buildpacks/lifecycle
 
 require (
-	github.com/BurntSushi/toml v1.1.0
+	github.com/BurntSushi/toml v1.2.0
```

Comment:
In an effort to fix clashing dependencies, I updated them all, and this is breaking the build. Will downgrade...